Library/src/main/java/com/shopify/testify/ScreenshotRule.kt

```diff
@@ -367,6 +378,10 @@ open class ScreenshotRule<T : Activity> @JvmOverloads constructor(
         }
     }
 
+    private fun applyOutputFileExists(screenshotUtility: ScreenshotUtility, outputFileName: String) {
```

Comment:
I think it would be cleaner to move this to `ScreenshotUtility`. e.g. `screenshotUtility.outputFileExists()`

 Then, I don't think you need to cache this value and you can just call it directly on Line 406 when you run the `assertFalse`.  Perhaps I'm missing something, but that seems to be sufficient.

Reply:
The latest commit addresses this feedback - let me know what you think